core/build.gradle

```diff
@@ -6,6 +6,9 @@ dependencies {
     compile project(':abi'),
             project(':crypto'),
             project(':tuples'),
+            project(':compat'),
+            "net.sourceforge.streamsupport:streamsupport:1.7.0",
+            "net.sourceforge.streamsupport:streamsupport-cfuture:1.7.0",
```

Comment:
@serso , can you please externalise the version number into a property?

Reply:
Fixed here 10487e6b94578cb62317a141769df95037bdf452